src/model/cloud-runner/workflows/build-automation-workflow.ts

```diff
@@ -65,4 +60,62 @@ export class BuildAutomationWorkflow implements WorkflowInterface {
       throw error;
     }
   }
+
+  private static get FullWorkflow() {
+    const setupHooks = CloudRunnerBuildCommandProcessor.getHooks(CloudRunner.buildParameters.customJobHooks).filter(
+      (x) => x.step.includes(`setup`),
+    );
+    const buildHooks = CloudRunnerBuildCommandProcessor.getHooks(CloudRunner.buildParameters.customJobHooks).filter(
+      (x) => x.step.includes(`build`),
+    );
+    const builderPath = path.join(CloudRunnerFolders.builderPathFull, 'dist', `index.js`).replace(/\\/g, `/`);
+    return `apt-get update > /dev/null
+      apt-get install -y zip tree npm git-lfs jq unzip git > /dev/null
+      npm install -g n > /dev/null
+      n stable > /dev/null
+      ${setupHooks.filter((x) => x.hook.includes(`before`)).map((x) => x.commands) || ' '}
+      export GITHUB_WORKSPACE="${CloudRunnerFolders.repoPathFull.replace(/\\/g, `/`)}"
+      ${BuildAutomationWorkflow.SetupCommands(builderPath)}
+      ${setupHooks.filter((x) => x.hook.includes(`after`)).map((x) => x.commands) || ' '}
+      ${buildHooks.filter((x) => x.hook.includes(`before`)).map((x) => x.commands) || ' '}
+      ${BuildAutomationWorkflow.BuildCommands(builderPath, CloudRunner.buildParameters.buildGuid)}
+      ${buildHooks.filter((x) => x.hook.includes(`after`)).map((x) => x.commands) || ' '}`;
+  }
+
+  private static SetupCommands(builderPath) {
```

Comment:
Note that in JavaScript and TypeScript the convention is to make all methods, variables, constants etc. `camelCase`.

We never use PascalCase except when using a `model class` or perhaps some kind of unit describing an `entity` or it's data directly.

Reply:
Whatever I have been doing most that day comes with me sometimes. 😁🙂

Reply:
Haha completely understand of course!

I'll set up some ESLint rules to mitigate this problem.